src/appsignal/cli.py

```diff
@@ -1,2 +1,118 @@
+from abc import ABC, abstractmethod
+import os
+
+from typing import Optional
+from docopt import docopt
+
+from .__about__ import __version__
+
+DOC = """AppSignal for Python CLI.
+
+Usage:
+  appsignal install [--push-api-key=<key>]
+  appsignal (-h | --help)
+  appsignal --version
+
+Options:
+  -h --help             Show this screen and exit.
+  --version             Show version number and exit.
+  --push-api-key=<key>  Push API Key to use for the installation.
+"""
+
+
 def run():
-    print("Hello AppSignal!")
+    version = f"AppSignal for Python v{__version__}"
+
+    arguments = docopt(DOC, version=version)
+
+    return command_for(arguments).run()
+
+
+class CLICommand(ABC):
+    @abstractmethod
+    def run(self) -> int:
+        raise NotImplementedError
+
+
+def command_for(arguments) -> CLICommand:
+    if arguments["install"]:
+        return InstallCommand(push_api_key=arguments["--push-api-key"])
+    else:
+        raise NotImplementedError
+
+
+INSTALL_FILE_TEMPLATE = """from appsignal import Appsignal
+
+appsignal = Appsignal(
+    active=True,
+    name="{name}",
+    push_api_key="{push_api_key}",
+    environment="development",
+)
+"""
+
+INSTALL_FILE_NAME = "appsignal_config.py"
```

Comment:
I have to look into this, because I would like to call it `appsignal.py` instead of `appsignal_config.py` as well.

But this file needs to import the `appsignal` *module*, the package, to do `from appsignal import Appsignal`. Because this file is also named `appsignal`, it shadows the module, and the file tries to import itself. 🤦 

But, relative imports, right? Well, you can only use those in a module context -- meaning you can't use them from Django's `manage.py`, as we do in the test setup.

There has to be a solution for this, I'll take a look.